website/src/components/broadcast/PlayerHero.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="hero-card d-flex flex-column flex-center" v-if="hero">
+    <div class="hero-name d-flex flex-row text-center font-weight-bold"><span v-if="roleIcon" class="icon" v-html="roleIcon"></span> {{heroName}}</div>
+    <div class="hero-role d-flex flex-row text-center">{{heroRole}}</div>
+    <div class="hero-portrait d-flex flex-row flex-center"><img :src="heroPortraitURL"/></div>
```

Comment:
You'd want something here if there's no image so you don't get the dead image icon.

```suggestion
    <div class="hero-portrait d-flex flex-row flex-center"><img v-if="heroPortraitURL" :src="heroPortraitURL"/></div>
```
Might work if you want to keep the side space (though empty). Otherwise you could have the `v-if` on the `hero-portrait` itself.

To be fair, the likelihood of having a Hero that doesn't have an image is low, but not impossible.

Reply:
done here af0f00d